client/my-sites/upgrades/domain-management/dns/dns-list.jsx

```diff
@@ -43,7 +43,11 @@ const DnsList = React.createClass( {
 	addDns: function( record ) {
 		addDnsAction( this.props.selectedDomainName, record, ( error ) => {
 			if ( error ) {
-				notices.error( error.message );
+				notices.error( error.message || this.translate( 'The DNS record could not be restored.' ) );
+			} else {
+				this.props.successNotice( this.translate( 'The DNS record has been restored.' ), {
+					duration: 5000
```

Comment:
Hmm, there might have been a slight misunderstanding when I pointed the same to @artpi in the [comment](https://github.com/Automattic/wp-calypso/pull/1916/files#r51361230) I linked earlier. I think he thought I meant all the components in `/domain-management` while I was pointing out the same thing you are here. @artpi, can we use `errorNotice` here? Since `successNotice` is obviously working...